README.md

```diff
@@ -55,6 +56,23 @@ USAGE:
 OPTIONS:
    --json-value, -j  Treat value as json
 
+
+---
+
+NAME:
+   meta lastSuccessfulMeta - Fetch lastSuccessfulMeta from an external job
```

Comment:
this is better integrated with existing `meta get` syntax and expose this as a flag.

I would rather have `meta get --last-successful job`

Reply:
Agree with Jithin. I think it will be clearer. It will implicitly imply that this is a `get` behavior and it doesn't allow `set`.

Reply:
Done